sample-configs/nri-resource-policy-configmap.example.yaml

```diff
@@ -0,0 +1,302 @@
+#
+# This example creates 3 ConfigMaps:
+#  - nri-resmgr-config.default: the default configuration
+#  - nri-resmgr-config.group.foo: the configuration for nodes in group foo
+#  - nri-resmgr-config.node.cl0-slave1: the configuration for node cl0-slave1
+#
+# You can assign nodes to group foo using the command
+#   kubectl label --overwrite node $NODE_NAME nri-resmgr.intel.com/group=foo
+#
+# You can remove nodes from group foo using the command
+#   kubectl label node $NODE_NAME nri-resmgr.intel.com/group-
+#
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nri-resmgr-config.default
```

Comment:
I'd suggest to simplify this by dropping the group and node config below and just add comments/notes here that the user can also specify per-node or per-group configs

```yaml
## Node-specific configuration can be specified with
#name: nri-resmgr-config.node.<node-name>

## Configuration specific to a group of nodes can be specified with
#name: nri-resmgr-config.group.<node-group-label-name>
```